.travis.yml

```diff
@@ -1,29 +1,17 @@
 language: php
 
-dist: trusty
-
-matrix:
-  fast_finish: true
-  include:
-    - env: DM=~6.0
-      php: 7.2
-    - env: DM=@dev
-      php: 5.5
-    - env: DM=~5.0
-      php: 5.6
-    - env: DM=~4.2
-      php: 7
-  allow_failures:
-    - env: DM=@dev
+php:
+  - '7.2'
+  - '7.3'
 
 sudo: false
 
-install: travis_retry composer require "wikibase/data-model=$DM" --prefer-source
+install: travis_retry composer install
 
 script: composer ci
 
 after_success:
-  - if [[ "`phpenv version-name`" != "7.1" ]]; then exit 0; fi
+  - if [[ "`phpenv version-name`" != "7.3" ]]; then exit 0; fi
```

Comment:
7.2?

Reply:
Doesn't really matter as long as a single currently run version is used

Reply:
Indeed. Though higher is normally a little bit faster.

Reply:
Done here https://github.com/wmde/WikibaseDataModelSerialization/commit/2d1a4732a329d70fabbb09337e4d59924f6ef0bd and in all of our other libraries